plotting_tools/PlotHelpers.h

```diff
@@ -0,0 +1,222 @@
+TH1D* get_hist_with_same_width_bins(TH1D* input_hist, string input_hist_name, vector<int> lxy_bins, float rebin, float int_lumi, float cross_sec, float BR, float N_tot)
+{
+  int n_bins = lxy_bins.size();
+  TH1D* hist = new TH1D((input_hist_name+"_1widthbins").c_str(), (input_hist_name+"_1widthbins").c_str(), n_bins-1, 0, n_bins-1);
+  for(int i=0; i<n_bins-1; i++)
+  {
+    hist->SetBinContent(i+1, input_hist->GetBinContent(i+1));
+  }
+  hist->GetXaxis()->SetLabelSize(0.04);
+  return hist;
+}
+
+TH1D* get_lxy_with_outer_tracker(TH1D* input_hist, string input_hist_name, vector<int> lxy_bins, float x_max, float rebin, float int_lumi, float cross_sec, float BR, float N_tot)
+{
+  float binListExtended[8] = {0, 2, 10, 24, 31, 70, 110, x_max}; // mm
+  int n_bins_max = 8;
```

Comment:
You could move it before `binListExtended`, make it `const int` and use to define the size of `binListExtended` (one less hardcoded number).